### ISSUE TITLE
Do we have a folder for baseline images where the images are not changed every run?

### ISSUE THREAD

**Issue by faith-berroya** [FRSOURCE/cypress-plugin-visual-regression-diff#87]
The plugin seems to compare the images from previous run instead of a baseline image. Do we have a setting for which image should we compare the current run? It's getting the wrong images for me as the baseline. 
Correct:
<img width="1631" alt="image" src="https://user-images.githubusercontent.com/101372793/184791471-85d1ca2b-4d21-4c11-9a10-c45aaa4099f9.png">
Wrong: (What the plugin gets, and pass)
<img width="1631" alt="image" src="https://user-images.githubusercontent.com/101372793/184791493-10b45a78-fa6c-4baa-9623-58142326db32.png">

**Comment by faith-berroya** [comment]
Hi @dhulme I'm confused as to whether the screenshots are fixed in the `image_snapshots` folder whenever a CI runs a new job. Is it changing ever new run? Or we only need 1 job run and it will remain the same for all succeeding jobs?

**Comment by FRSgit** [comment]
Hi @faith-berroya!

Thank you for using this plugin.

This is the proposed usage flow:
1. you write test suite using `cy.get('element').matchImage()` syntax (notice that `updateImages` is not used here),
2. the images get during the first test run,
3. on subsequent test runs newly taken screenshots are compared against images from step 2,
4. if you need to update images from step 2 you should cypress tests with a command `npx cypress run --env "pluginVisualRegressionUpdateImages=true"` (notice `pluginVisualRegressionUpdateImages` flag - it makes plugin update images on the hard drive).

To sum up - `updateImages` option (whether it is set via command configuration object or cypress env) makes plugin update images stored in your project directory and should be used **only** when you want to explicitly update images.

**Comment by FRSgit** [comment]
Closing the issue as it was resolved by the comment above and the discussion thread here #88
